app/views/users/sessions/new.html.erb

```diff
@@ -1,3 +1,5 @@
 <% title t('landing.page_title') %>
 
-<%= govuk_start_button(text: t('calls_to_action.sign_in'), href: user_azure_ad_omniauth_authorize_path, as_button: true) %>
+<% unless flash.first&.last&.include?("You cannot access this service") %>
```

Comment:
I think we need to re-think how we handle errors depending on users authentication state etc.